apps/atoms/atom.cpp

```diff
@@ -791,18 +821,18 @@ generate_atom_file(Free_atom& a, double core_cutoff, const std::string& lo_type,
 
     //            fprintf(fout, "        <lo l=\"%i\">\n", l);
```

Comment:
unless there is a reason to keep this, I would suppress the commented code entirely. We can always put it back with git when necessary